neutron/templates/configmaps/ml2-conf.ini.yaml.j2

```diff
@@ -19,9 +25,14 @@ data:
     vxlan_group = 239.1.1.1
 
     [securitygroup]
+    {% if ironic.enabled %}
+    firewall_driver = neutron.agent.linux.iptables_firewall.OVSHybridIptablesFirewallDriver
```

Comment:
make sure that neutron.agent.linux.iptables_firewall.OVSHybridIptablesFirewallDriver is still valid.